runtime/src/builtin/network.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use fvm_shared::clock::EPOCH_DURATION_SECONDS;
-pub use fvm_shared::BLOCKS_PER_EPOCH as EXPECTED_LEADERS_PER_EPOCH;
-
+pub const EPOCH_DURATION_SECONDS: i64 = 30;
```

Comment:
Commentary: again _assumed_ duration, but isn't something that gets defined by the actors.

Reply:
Yes, but changing this would require a massive migration so I think it's OK if the actors assume a specific duration. But I'll comment on that.